lib/wor/paginate/matchers/rspec.rb

```diff
@@ -0,0 +1,14 @@
+require_relative 'formatter_matcher'
+
+RSpec::Matchers.define :be_paginated_with do |formatter|
+  define_matcher_for self, formatter
+end
+
+RSpec::Matchers.define :be_paginated do
+  define_matcher_for self, Wor::Paginate::Formatter
+end
+
+def define_matcher_for(matcher, formatter)
+  formatter_matcher = Wor::Paginate::Matchers::FormatterMatcher.new(formatter)
```

Comment:
I don't know we I cant just call FormatterMatcher just like I did with the MockedAdapter inside the FormatterMatcher class. Just tell me if you know a way to do that